app/src/main/java/org/bostwickenator/ftpuploader/WifiSetupActivity.java

```diff
@@ -123,7 +123,6 @@ private void updateWifiSwitch() {
                         summary = res.getString(R.string.connectionStateConnecting);
                         break;
                     default:
-                        res.getString(0, "test");
```

Comment:
Patch from Bostwickenator/STGUploader@5a6681f should be in it's own PR

Reply:
Would be better, true.